moduleroot/.github/workflows/cron.yml.erb

```diff
@@ -0,0 +1,93 @@
+name: Nightly tests
+
+on:
+  schedule:
+    - cron: '4 * * * *'
```

Comment:
Do we want to limit this to the master branch? Otherwise stable branches will end up running for eternity.

Reply:
Answering my own question: looks like schedule only works on the base branch if my understanding of https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#scheduled-events is correct.

Reply:
Why do we want this at all? How does it report? And what do we do with the information?

Reply:
So we already have Travis, which emails (though I must admit I didn't see one from https://travis-ci.org/github/theforeman/puppet-pulp/jobs/739154429 last night). At least when I got those failure notification I checked why it they failed.

I must also admit that I don't know how Github Actions does its notification, but perhaps it's just something we need to try?

Reply:
Ok, I can be willing to try it. And revisit. Trying to balance the amount of concurrency we get with ensuring we have jobs that provide value and provide notifications that we can/do take action on. Will this mean that all our puppet modules kick off a cron job at the same time every day?

Reply:
It does. I thought about limiting concurrency but perhaps we should first see how many issues we'll see. For what it's worth, only the default branch is tested in cron. That means the stable branches aren't tested.